tutorials/roostats/rs101_limitexample.py

```diff
@@ -0,0 +1,186 @@
+## \file
+## \ingroup tutorial_roostats
+## \notebook
+## Limits: number counting experiment with uncertainty on both the background rate and signal efficiency.
+##
+## The usage of a Confidence Interval Calculator to set a limit on the signal is illustrated
+##
+## \macro_image
+## \macro_output
+## \macro_code
+##
+## \author Kyle Cranmer
+
+import ROOT
+
+# --------------------------------------
+# An example of setting a limit in a number counting experiment with uncertainty on background and signal
+
+# to time the macro
+t = ROOT.TStopwatch()
+t.Start()
+
+# --------------------------------------
+# The Model building stage
+# --------------------------------------
+wspace = ROOT.RooWorkspace()
+wspace.factory("Poisson::countingModel(obs[150,0,300], "
+               "sum(s[50,0,120]*ratioSigEff[1.,0,3.],b[100]*ratioBkgEff[1.,0.,3.]))") # counting model
+# wspace.factory("Gaussian::sigConstraint(ratioSigEff,1,0.05)") # 5% signal efficiency uncertainty
+# wspace.factory("Gaussian::bkgConstraint(ratioBkgEff,1,0.1)") #  10% background efficiency uncertainty
+wspace.factory("Gaussian::sigConstraint(gSigEff[1,0,3],ratioSigEff,0.05)") # 5% signal efficiency uncertainty
+wspace.factory("Gaussian::bkgConstraint(gSigBkg[1,0,3],ratioBkgEff,0.2)") # 10% background efficiency uncertainty
+wspace.factory("PROD::modelWithConstraints(countingModel,sigConstraint,bkgConstraint)") # product of terms
+wspace.Print()
+
+modelWithConstraints = wspace.pdf("modelWithConstraints") # get the model
+obs = wspace.var("obs") # get the observable
```

Comment:
```suggestion
modelWithConstraints = wspace["modelWithConstraints"] # get the model
obs = wspace["obs"] # get the observable
```

In PyRoot, you can universally get anything from a RooWorkspace with the `[]`-operator. You should use this everytime you see `var()`, `pdf()`, or `function()` in C++ (there are also more occurences of `var()` to replace in the lines below).